### ISSUE TITLE
Creating a Project from Cluster Explorer creates a namespace with the same name

### ISSUE THREAD

**Issue by izaac** [rancher/dashboard#3090]
Rancher version: `2.6 master-head (05/25/2021)` _f90b36e_

- Create a cluster
- Navigate to the cluster and click on `Cluster -> Project/Namespaces`
- Click the `Create` button to create a Project

Result:

The Project is created but this also creates a namespace of the same name.
This behavior wasn't previously observed on Cluster Manager

Where `test3` is a Project created from `Cluster Explorer` this is its associated namespace describe

```
k describe ns test3
Name:         test3
Labels:       <none>
Annotations:  cattle.io/status:
                {"Conditions":[{"Type":"ResourceQuotaInit","Status":"True","Message":"","LastUpdateTime":"2021-05-25T21:19:18Z"},{"Type":"InitialRolesPopu...
              lifecycle.cattle.io/create.namespace-auth: true
              management.cattle.io/system-namespace: true
Status:       Active

No resource quota.

No LimitRange resource.
```

This is how it looks from `Cluster Manager`

The project name in the `Cluster Explorer` UI is following the following format `(cluster_name:project_name)`

![Screen Shot 2021-05-25 at 2 25 17 PM](https://user-images.githubusercontent.com/27659/119570529-19f89480-bd65-11eb-9d6b-ce75abc57f72.png)

Related issue: https://github.com/rancher/dashboard/issues/2500

**Comment by vincent99** [comment]
A project is (and has been, including Ember) a CRD (to define it) and a namespace (to hold things that are "in" the project), that both exist in the local cluster.  Are you looking at the namespaces of the local cluster instead of in the cluster the project is for?

**Comment by izaac** [comment]
@vincent99 I created a project from Ember too with name `test2` But I don't see a namespace associated with it on the `local` cluster

```
k describe ns test2
Error from server (NotFound): namespaces "test2" not found
```

Aren't we seeking parity here?

I am checking this from `local` so far as the namespaces are [not getting grouped](https://github.com/rancher/dashboard/issues/2500#issuecomment-848267026) on downstream clusters.

**Comment by vincent99** [comment]
@codyrancher Yeah there's stuff about setting the creatorId also.  You should use the norman API for create (other operations can use it too if you want to be consistent)

**Comment by codyrancher** [comment]
I'll switch over.

**Comment by izaac** [comment]
Rancher version `2.6 master-head (05/07/2021)` _b2f8824_

The Project creation is identical compared with Ember now. Even the same labels/annotations are applied in a comparative test with Ember.
I was able to clone a project. Namespaces are left intact and a new cloned and empty project is created,
I was able to remove empty projects and with namespaces. Namespaces are moved to `Not in Project`

Separate issue opened: https://github.com/rancher/dashboard/issues/3175
